src/plugins/saved_objects_finder/README.md

```diff
@@ -0,0 +1,5 @@
+# `savedObjectsFinder` plugin
+
+NOTE: This plugin is deprecated and will be removed in 8.0. See https://github.com/elastic/kibana/issues/46435 for more information.
```

Comment:
I don't thing the deprecation of the `saved_objects` plugin can also be applied to UI components, see 

_The remaining dependencies should be limited to the SavedObjectClass and the UI components like SavedObjectSaveModal, SavedObjectFinder and misc helpers. These should either be maintained by the wider Kibana analytics team (the main consumer) or shared UX team._

https://github.com/elastic/kibana/issues/46435#issuecomment-1038898869

So I think we don't already need to deprecate this new plugin with a Time Machine (since 8.0 is already out)

Reply:
Sounds good, I'll remove this deprecation notice.